shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -281,6 +287,18 @@ bool FlutterWindowsEngine::RunWithEntrypoint(const char* entrypoint) {
   return true;
 }
 
+void FlutterWindowsEngine::OnVsync(intptr_t baton) {
+  auto current_time = embedder_api_.GetCurrentTime();
+  if (view_ == nullptr) {
+    embedder_api_.OnVsync(engine_, baton, current_time,
```

Comment:
~~Doesn't the embedder. Have to wait till the start time to invoke the callback?~~

Nope, not since https://github.com/flutter/engine/commit/b5f59ed89d